native/cocos/math/Utils.h

```diff
@@ -39,10 +39,15 @@ bool equals(F a, F b) {
 template <typename F>
 bool approx(F a, F b, F maxDiff) {
     static_assert(std::is_floating_point<F>::value, "number expected");
-    maxDiff = maxDiff || EPSILON;
     return std::abs(a - b) <= maxDiff;
 }
 
+template <typename F>
+bool approx(F a, F b) {
+    static_assert(std::is_floating_point<F>::value, "number expected");
+    return std::abs(a - b) <= EPSILON;
```

Comment:
Should use `std::fabs()` as https://github.com/cocos/cocos-engine/pull/12075 did.

Reply:
DONE